changelog.txt

```diff
@@ -13,6 +13,8 @@ V1.XX.X
 [QMS-165] OSX: GIS files not loaded when clicked
 [QMS-173] Misleading help text for DEM range slider
 [QMS-178] Automate list of code contributors in about dialog
+[QMS-158] Change Routino search for [prefix-]profiles.xml.
+  Searches the routino database path (user defined and writable) and then default global location (not user writable). This behaviour mirrors the commandline routino-router program, that is dbpath, prefix and "profiles.xml" combinations will be tried, if none exist then the file '/usr/local/share/routino/profiles.xml' (or custom installation location) will be used. This allows profiles to be customized for local speeds or vehicle length and weight etc linked to specific map areas.
```

Comment:
Sorry I had corrected this - not sure how this went through.

Reply:
Question - Do you only want a single line in changelog?  I understood that you want user changes explained!  Else where do I  explain that user prefix-profiles.xml files can be created in the database directory.

Reply:
Just the ticket title. Interested users can find the ticket by the number to read the details. And  it would be nice if you document the new behavior in the Wiki once the code is merged.

Btw why did you close the PR if you still intent to get the code merged?